app/_locales/en/messages.json

```diff
@@ -1,14 +1,14 @@
 {
   "appName": {
-    "message": "Behave!",
+    "message": "Behave! — Offensive Comments Blocker",
```

Comment:
Lucas, we can't make title longer, as there is a 45 characters limit which will not fit your suggestions. I suggest we brainstorm later and most likely come up with something better.